### DISCUSSION TITLE
Logout after writing a prompt

### DISCUSSION THREAD

**Discussion by Mte90** [TabbyML/tabby#3498] [General]
**Describe the bug**
I have tabby at autostart of my workstation, it remember by yesterday login but if I wrote a prompt in the chat panel the AI starts replying to me but I am getting logged out.
When I login the chat I started is missing and I have to do it from start the prompt.
Sometimes I get also an alert during the login about the database locked so I have to close the tabby process and restart it.

**Information about your version**
0.21-rc4

**Comment by wsxiaoys** [comment]
Hi - since Tabby utilize your personal auth token for chat panel / server communication, could you verify if the auth token changed during your restart? If not, can you share the output panel logs when you getting logout?

**Comment by Mte90** [comment]
So I checked (it is happened right now), from the avatar menu the token before writing in the chat panel and after the login it is the same.

**Comment by Mte90** [comment]
![image](https://github.com/user-attachments/assets/f6b7e3fe-436d-4b8b-86d6-53027c8ce77a)

This is what I get after login, use it a bit and I get a logout again.

**Comment by wsxiaoys** [comment]
Can you share your auto-start setup for Tabby?

**Comment by Mte90** [comment]
I have this script on auto start in KDE on Linux: `/[path]/tabby_x86_64-manylinux2014-cuda122/tabby serve --model Qwen2.5-Coder-7B --chat-model Mistral-7B --device cuda --port 5000 &`

**Comment by wsxiaoys** [comment]
My best guess is that your setup initiates multiple Tabby processes, which in turn causes the SQLite database to become locked. I recommend configuring a supervisor to prevent such situations.

**Comment by Mte90** [comment]
I can confirm that there is only one tabby process as I have just one in the process manager when I have to kill to restart it.

**Comment by wsxiaoys** [comment]
Depending on your process termination setup, you might not be waiting for the process to exit gracefully, which could result in the SQLite database file being locked.

check https://stackoverflow.com/questions/151026/how-do-i-unlock-an-sqlite-database for a potential workaround

**Comment by Mte90** [comment]
What is the path for the database? So I can check when happens again.

**Reply by Mte90** [reply]
it is very annoying as I get various times the logout also after that tabby started replying.

**Reply by wsxiaoys** [reply]
the default path for db is `~/.tabby/ee/db.sqlite`

**Reply by Mte90** [reply]
So I got a logout after asking in the chat panel, and did the login and again the db is locked.
I waited a while and I was able to login (I forgot to login again).

I think that this logout is something browser side that fails not server side. It shouldn't redirect to to the login if I was able to write on the chat panel and get an answer.

**Comment by Mte90** [comment]
I see a `db.sqlite-journal` file generated but appears/disappear.
I think because it is still processing the chat reply and saving it in the meantime that I am trying again.

**Reply by Mte90** [reply]
it is very annoying this automatic logout, I am now reading the sqlite database to get the output.

**Reply by wsxiaoys** [reply]
To assist with debugging, please provide your configuration (as detail as possible) to help us reproduce the behavior. At the moment, we are unable to reproduce it.

**Reply by Mte90** [reply]
I already shared the command I have in autorun on my machine, what else I can share and how?

**Reply by Mte90** [reply]
just a ping, because this is a very annoying problem.
